sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -860,9 +872,10 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
       response.parsedBody == undefined ||
       (Array.isArray(response.parsedBody) && response.parsedBody.length == 0)
     ) {
+      response.status = 404;
```

Comment:
According to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClient.cs#L920 `HttpStatusCode.NoContent` is considered as Message Entity Not Found. Where as we are specifically checking for `response.parsedBody` to be undefined or an empty array.

Any reason we cannot look for `HttpStatusCode.NoContent` i.e 204 status code?